### DISCUSSION TITLE
Dynamic Href Error when using Pages Routes

### DISCUSSION THREAD

**Discussion by allindeveloper** [vercel/next.js#65559] [Help]
### Summary

I get an error when using nextjs 13.4.6

> Dynamic href `/[store-locale]/[category]/[subCategory]` found in <Link> while using the `/app` router, this is not supported. Read more: https://nextjs.org/docs/messages/app-dir-dynamic-href

and my project uses `pages` routing
the docs says this error shows only in app routes, so i am confused what might be wrong

**Comment by eyesfocus** [comment]
Same here with next 14.1.1
